Wikipedia/Code/UIView+WMFFadeInAndUp.m

```diff
@@ -0,0 +1,20 @@
+#import "UIView+WMFFadeInAndUp.h"
+
+@implementation UIView (WMFFadeInAndUp)
+
+- (void)wmf_zeroLayerOpacity {
+    self.layer.opacity = 0.0;
+}
+
+- (void)wmf_fadeInAndUpCompletion:(void (^ __nullable)(BOOL finished))completion {
```

Comment:
I may not need this completion block for the follow-on animation ticket. If not I'll remove it.